dora/core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -926,7 +927,7 @@ public List<BlockWorkerInfo> getLostWorkers() throws IOException {
     try (ReinitBlockerResource r = blockReinit()) {
       // Use membership mgr
       if (mMembershipManager != null && !(mMembershipManager instanceof MasterMembershipManager)) {
-        return mMembershipManager.getFailedMembers().stream()
+        return Streams.stream(mMembershipManager.getFailedMembers())
```

Comment:
so snapshot() isnt' used in current code base is it? then why do we need snapshot in the first place?

Reply:
Snapshots are immutable and static view objects, the workers in a snapshot does not change over time and does not reflect the state of the cluster in real time. They are useful for caching the state for an extended period of time, and is equivalent to what we have as `List<WorkerInfo>` currently in the return position of `get*Members()`.

What's proposed in this PR is dynamic, live view objects which always pulls the latest state of the cluster.

Reply:
removed live views. only snapshots are returned by membership manager.